docs/rules/one-var-declaration-per-line.md

```diff
@@ -0,0 +1,85 @@
+# Require or disallow an newline around variable declarations (one-var-declaration-per-line)
+
+Some developers declare multiple var statements on the same line:
+
+```js
+var foo, bar, baz = 0;
```

Comment:
The descriptions below sound like this rule can be used to enforce this style as well.
However, the examples for both "always" and "initializations" show that this style would be considered an error in both.